sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -57,7 +60,7 @@ case class CreateDatabase(
       CatalogDatabase(
         databaseName,
         comment.getOrElse(""),
-        path.getOrElse(catalog.getDefaultDBPath(databaseName)),
+        catalog.createDatabasePath(databaseName, path),
```

Comment:
oh, we have `defaultTablePath` in SessionCatalog. Let's use `defaultDatabasePath` as the better. What do you think?